Tasks/VsTest/vstest.ts

```diff
@@ -1533,3 +1490,17 @@ function responseContainsNoTests(filePath: string): Q.Promise<boolean> {
         }
     });
 }
+
+function isNullOrWhitespace(input) {
+    if (typeof input === 'undefined' || input == null) {
+        return true;
+    }
+    return input.replace(/\s/g, '').length < 1;
+}
+
+function getTaskMatchOption(): any {
```

Comment:
since this is a new major version and the goal is consistency across in-the-box tasks i think it would be better to go with `matchBase:false`.

the behavior of matchBase is achievable by prepending the file name with `**\`, so `matchBase:false` doesn't limit functionality.